core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingAdminService.scala

```diff
@@ -0,0 +1,45 @@
+package org.broadinstitute.dsde.rawls.billing
+
+import akka.http.scaladsl.model.StatusCodes
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.rawls.RawlsExceptionWithErrorReport
+import org.broadinstitute.dsde.rawls.dataaccess.SamDAO
+import org.broadinstitute.dsde.rawls.model.{
+  BillingProjectAdminResponse,
+  ErrorReport,
+  RawlsBillingProjectName,
+  RawlsRequestContext,
+  SamResourceTypeAdminActions,
+  SamResourceTypeNames
+}
+import org.broadinstitute.dsde.rawls.workspace.WorkspaceRepository
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class BillingAdminService(samDAO: SamDAO,
+                          billingRepository: BillingRepository,
+                          workspaceRepository: WorkspaceRepository,
+                          ctx: RawlsRequestContext
+)(implicit protected val ec: ExecutionContext)
+    extends LazyLogging {
+
+  def getBillingProject(billingProjectName: RawlsBillingProjectName): Future[BillingProjectAdminResponse] =
```

Comment:
Maybe `getBillingProjectAndWorkspaces`?

Reply:
Happy to change it, but I do want it to be clear that the full workspaces aren't included which I'm not sure it is with `getBillingProjectAndWorkspaces`.

Could do something like `getBillingProjectSupportSummary` which is how it's worded in Sam for managed groups

Reply:
`getBillingProjectSupportSummary` seems reasonable.